jenkins/job_manager/script/SettingsParser.groovy

```diff
@@ -0,0 +1,53 @@
+import org.yaml.snakeyaml.Yaml
+import groovy.mock.interceptor.MockFor
+
+class SettingsParser {
+    String yaml
+    Map config
+    List<Settings> settingsList = []
+
+    static Logger loggerMock
```

Comment:
mock is something used for testing http://stackoverflow.com/questions/2665812/what-is-mocking . Please remove word mock from variable name